### ISSUE TITLE
Some RFD interlinking is broken

### ISSUE THREAD

**Issue by Keno** [oxidecomputer/rfd-api#104]
I hope this is the correct place to post this. I was idly browsing the RFD pages, when I noticed that a number of inter-RFD links are broken. For example, in https://rfd.shared.oxide.computer/rfd/0024, in section 4.4.3 (https://rfd.shared.oxide.computer/rfd/0024#_networking), there is a link to RFD21, but the link target is https://github.com/oxidecomputer/rfd/tree/master/rfd/0021 rather than https://rfd.shared.oxide.computer/rfd/0021. Note that it does have a hoverover to show metadata, so some part of the pipeline is clearly recognizing this.

**Comment by benjaminleonard** [comment]
Many RFDs, especially if they predate the current RFD site will link to the documents on Github instead (which you would not have access to).

The hover preview uses a slightly cursed regex to discover any RFD links, and in theory we could preprocess the document to change all of these to RFD site links. I think we'd eventually link a linter to catch this stuff, along with some other formatting issues that crop up from time to time.

Good catch, and thanks for reporting.

**Comment by augustuswm** [comment]
Agreed, it would be good to have a processor step that rewrites urls like these. I'll look at putting something like that together.
